server/app/views/BaseHtmlView.java

```diff
@@ -146,6 +230,52 @@ protected DivTag renderPaginationDiv(
     return div.with(br());
   }
 
+  private LiTag renderPaginationPageButton(
+      int page, boolean isCurrentPage, Function<Integer, Call> linkForPage) {
+    return li().withClass("usa-pagination__item usa-pagination__page-no")
+        .with(
+            a(Integer.toString(page))
+                .withClass("usa-pagination__button")
+                .withCondClass(isCurrentPage, "usa-pagination__button usa-current")
+                .withHref(linkForPage.apply(page).url())
+                .attr("aria-label", "Page" + page)
+                .condAttr(isCurrentPage, "aria-current", "page"));
+  }
+
+  private LiTag renderPreviousPageButton(int page, Function<Integer, Call> linkForPage) {
+    return li().withClass("usa-pagination__item usa-pagination__arrow")
+        .with(
+            a().withClass("usa-pagination__link usa-pagination__previous-page")
+                .attr("aria-label", "Previous page")
+                .withHref(linkForPage.apply(page - 1).url())
+                .with(
+                    Icons.svg(Icons.NAVIGATE_BEFORE)
+                        .withClasses("usa-icon", "h-4", "w-4")
+                        .attr("role", "img")
+                        .attr("aria-hidden", "true"),
+                    span("Previous").withClass("usa-pagination__link-text")));
+  }
+
+  private LiTag renderNextPageButton(int page, Function<Integer, Call> linkForPage) {
+    return li().withClass("usa-pagination__item usa-pagination__arrow")
+        .with(
+            a().withClass("usa-pagination__link usa-pagination__next-page")
+                .attr("aria-label", "Next page")
+                .withHref(linkForPage.apply(page + 1).url())
+                .with(
+                    span("Next").withClass("usa-pagination__link-text"),
```

Comment:
Unfortunately, the reason they have to be in a different order is that, going from left to right, the icon renders first for the "Previous" section and the icon renders last for the "Next" section.  I think the easiest way is to show how it will need to appear visually:  
![image](https://github.com/civiform/civiform/assets/134556723/42fb9a04-dafb-4a08-8b8c-cbce422b7bc0)

Reply:
Oh duh! I'm silly 😄